javascript/apps/taiga/src/assets/i18n/workspace/en-US.json

```diff
@@ -50,7 +50,16 @@
       "removed_member": "{{fullName}} removed.",
       "undo": "Undo",
       "action_undone": "Action undone",
-      "no_longer_member": "You are no longer a member of {{workspace}} workspace"
+      "no_longer_member": "You are no longer a member of {{workspace}} workspace."
     }
+  },
+  "leave": {
+    "action": "Leave workspace",
+    "title": "Leave workspace title?",
```

Comment:
We should replace 'workspace title' with the real workspace title :stuck_out_tongue_closed_eyes: